lib/gateways/notification/govNotify.js

```diff
@@ -0,0 +1,28 @@
+module.exports = ({ client, log }) => {
+  function send(dropbox, requestedBy, requestedByEmail) {
+    try {
+      log.info('Sending email notification to', {
+        requestedBy
+      });
+
+      const personalisation = {
+        name: requestedBy,
+        residentName: dropbox.customerName,
+        description: dropbox.description,
+        linkToDropbox: `${process.env.URL_PREFIX}/dropboxes/${dropbox.id}/view`
+      };
+
+      return client
+        .sendEmail(process.env.GOV_NOTIFY_TEMPLATE_ID, requestedByEmail, {
```

Comment:
I think there is a "configuration" section in the dependencies file that could be used for these values, instead of relying directly on the `process.env` stuff here. Slightly easier to mock those values in the tests that way too as you can inject some dummy values straight in.